charts/windows-ad-setup/templates/credentialspecs.yaml

```diff
@@ -23,4 +23,31 @@ credspec:
     NetBiosName: {{ $.Values.activeDirectory.domain.NetBIOSName }}
   CmsPlugins:
     - ActiveDirectory
+---
+apiVersion: windows.k8s.io/v1
+kind: GMSACredentialSpec
+metadata:
+  name: {{ $suffix }}-ccg

```

Comment:
we can just create both since there's no benefit in introducing a flag. This is just a configuration resource anyways.